src/app/components/team/SmoochBot/SmoochBotTextEditor.js

```diff
@@ -15,7 +14,7 @@ const useStyles = makeStyles(theme => ({
 
 const ValueOrPlaceholder = ({ value, field, children }) => (value || !placeholders[field])
   ? children(value)
-  : <FormattedMessage {...placeholders[field]}>{children}</FormattedMessage>;
+  : <FormattedMessage {...placeholders[field]}>{children}</FormattedMessage>; // eslint-disable-line @calm/react-intl/missing-attribute
```

Comment:
Note to reviewers: I am not sure how to resolve this, or if its necessary to. Any guidance would be appreciated